controllers/PollController.ts

```diff
@@ -59,6 +67,79 @@ export default class PollController implements PollControllerI {
             .then((poll: Poll) => res.json(poll));
     }
 
+    /**
+     * Retrieves all polls from the database and returns an array of polls.
+     * @param {Request} req Represents request from client
+     * @param {Response} res Represents response to client, including the
+     * body formatted as JSON arrays containing the tuit objects
+     */
+    getAllPolls = (req: Request, res: Response) => {
+        PollController.pollDao.getAllPolls()
+            .then((poll: Poll[]) => res.json(poll));
+    }
+
+    /**
+     * Retrieves a poll from the database by id and returns a poll.
+     * @param {Request} req Represents request from client
+     * @param {Response} res Represents response to client, including the
+     * body formatted as JSON arrays containing the tuit objects
```

Comment:
body formatted as JSON arrays containing the poll objects

Reply:
Updated